scripts/setup-env.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable consistent-return, no-console */
+
+const fs = require('fs');
+const { exec } = require('child_process');
+const path = require('path');
+
+function updateConfig() {
+  if (process.env.ISSUER === undefined || process.env.CLIENT_ID === undefined) {
```

Comment:
Don't we also require `CLIENT_SECRET`, `USER_NAME`, and `PASSWORD`?

Reply:
Yes, we need them as well. Let me update it.